common/source/docs/common-gripper-servo.rst

```diff
@@ -55,4 +55,4 @@ The gripper position can be controlled during a mission in AUTO mode using the :
 Controlling Gripper from the ground station
 ===========================================
 
-The gripper can be opened or closed in real-time from the ground station if the ground station sends a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ with the command field set to DO_GRIPPER (i.e. 211) but there are no ground stations (as far as we know) that support doing this.
\ No newline at end of file
+The gripper can be opened or closed in real-time from the ground station if the ground station sends a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ with the command field set to DO_GRIPPER (i.e. 211) but there are no ground stations (as far as we know) that support doing this.
```

Comment:
this is no change?

Reply:
np

Reply:
I think the diff is in whitespace, one file ends on a character and the other has a newline at the end.